onnxscript/tools/memory_peak_test.py

```diff
@@ -0,0 +1,56 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import os
+import time
+import unittest
+
+import numpy as np
+import torch
+
+from onnxscript.tools.memory_peak import get_memory_rss, start_spying_on
+
+
+class TestMemoryPeak(unittest.TestCase):
```

Comment:
For some reason this test was not run in the CI it seems (judging from the lack of coverage). We need to understand what was going in.

Reply:
It is expected. The code is starting another process to monitor the memory. It cannot be done from the same process. The code coverage is not capture in the new process.